From 72ed428100ed4ec28d1d63ab08a10a44c14bfbcd Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Thu, 16 Nov 2017 17:28:57 +0800
Subject: [PATCH 2820/5242] MLK-16841-2: ARM64: dts: freescale: fsl-imx8mq:
 add 'fsl,ratio-1-1'

commit  931c268968988ac1b442a10460b231a9dfe920a9 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add fsl,ratio-1-1 into mscale B0 chip. Please remove this property if you
want to run A0 chip.

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index b54013b..73bb17b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -957,6 +957,7 @@
 		clock-names = "ipg", "ahb";
 		#dma-cells = <3>;
 		fsl,sdma-ram-script-name = "imx/sdma/sdma-imx7d.bin";
+		fsl,ratio-1-1;
 		status = "okay";
 	};
 
-- 
1.7.9.5

